src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -240,7 +240,9 @@ The .NET Foundation licenses this file to you under the MIT license.
       <IlcArg Include="@(TrimmerRootDescriptor->'--descriptor:%(FullPath)')" />
       <IlcArg Condition="'$(NativeLib)' != ''" Include="--nativelib" />
       <IlcArg Condition="'$(CustomNativeMain)' == 'true'" Include="--splitinit" />
-      <IlcArg Condition="$(ExportsFile) != ''" Include="--exportsfile:$(ExportsFile)" />
+      <IlcArg Condition="'$(ExportsFile)' != ''" Include="--exportsfile:$(ExportsFile)" />
+      <IlcArg Condition="'$(_targetOS)' != 'win' and '$(DebuggerSupport)' != 'false'" Include="--export-dynamic-symbol:DotNetRuntimeDebugHeader" />
```

Comment:
Can we use this line even on Windows instead of [this](https://github.com/dotnet/runtime/blob/e48ae1edf9e675f7edbec3009db13d0160061e47/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets#L332)?

Reply:
Yes, but there would need to be 2 options one for data exports and one for code like "--export-data-symbol" and "--export-code-symbol".  On Windows "DATA" needs to be added to the def file entry for exporting globals.